### DISCUSSION TITLE
Comparison with deepsoil

### DISCUSSION THREAD

**Discussion by geoport** [Caltech-geoquake/PySeismoSoil#13] [General]
Have you tried to compare  your nonlinear analysis results with deepsoil ? When I analyse one of your examples I got too different result from deepsoil. For example max of spectral acceleration on surface was 0.3 from pyseismo and 0.12 from deepsoil.

**Comment by jsh9** [comment]
Hi @geoport , 

> Have you tried to compare your nonlinear analysis results with deepsoil ? 

No.

We benchmarked our models against actual earthquake recordings (in [Shi & Asimaki, 2017](https://authors.library.caltech.edu/75704/)), which we believed to be of higher value than any numerical models (including our own).

In particular, Shi & Asimaki (2017) demonstrated that because our numerical model took into account soil strength, it was able to _**avoid under-predicting**_ ground motions (see Figure 15 in the paper).

My guess is: the discrepancy between our result and Deepsoil's may have come from Deepsoil under-predicting ground motions.

**Comment by jsh9** [comment]
I converted this issue into a discussion.  Because this is not a discussion about the quality of the software itself (such as bugs), but a discussion of the math and science under it.
